pylearn2/train_extensions/tests/test_wmape_channel.py

```diff
@@ -0,0 +1,67 @@
+"""
+Tests for WMAPE.
+"""
+from pylearn2.config import yaml_parse
+from pylearn2.testing.skip import skip_if_no_sklearn
+
+
+def test_wmape():
+    """Test WMapeChannel."""
+    skip_if_no_sklearn()
+    trainer = yaml_parse.load(test_yaml)
+    trainer.main_loop()
```

Comment:
Can you check here that, after training, the channels have been computed, and have the right value?